openapi.yaml

```diff
@@ -967,6 +967,28 @@ definitions:
         description: Credit hours passed for GPA calculation.
         type: integer
         example: 146
+      levelCode:
+        description: |
+         Academic level code for GPA calculation.
+         00: Level Not Declared
+         02: Graduate
+         03: Postbac Degree Seeking
+         01: Undergraduate
```

Comment:
Is there a reason they are ordered this way? Kind of bugs me that 01 doesn't come before 02 and 06 isn't after 05

Reply:
Haha, ok. I just paste whatever I got from the Banner table. I agree we should sort it and make it easier to read. Will fix.